bin/cli.js

```diff
@@ -90,35 +90,40 @@ mocha.setup({
   useColors: !opts['no-color']
 })
 
-try {
-  mocha.reporter(opts.reporter || 'spec')
-}
-catch (e) {
-  // opts.reporter is likely a path to a 3rd party reporter
-  if (typeof process === 'undefined') {
-    // a poor node.js process shim - totally not great
-    var sys = require('system')
-    this.process = {
-      pid: sys.pid,
-      env: sys.env,
-      argv: sys.args.splice(),
-      stdin: sys.stdin,
-      stdout: sys.stdout,
-      stderr: sys.stderr
+// fall back to spec by default
+var reporter = 'spec'
+
+if ( opts.reporter ) {
+  // check to see if it is a third party reporter
+  try {
+    // I don't want to use isAbsolute here as it could be a node module or a relative path
+    if (opts.reporter.indexOf('.') === 0) {
+      opts.reporter = fs.absolute(opts.reporter)
     }
+    reporter = require(opts.reporter)
+  } catch (e) {
+    reporter = opts.reporter
   }
+}
 
-  // I don't want to use isAbsolute here as it could be a node module or a relative path
-  if (opts.reporter.indexOf('.') === 0) {
-    opts.reporter = fs.absolute(opts.reporter)
+// phantomjs exits immediately if it can't find a module due to exitOnError: true from above... that should probably be false
+// either way a module may lazily require something and fail later, so a try/catch with an informative message isn't possible now.
+//
+// Remember that PhantomJS is not Node.js - the modules available to phantomjs are different than node's.
+// If you need access to built-in Mocha reporters, access them off of `Mocha.reporters`, like `Mocha.reporters.Base`.
+mocha.reporter(reporter)
+
+if (typeof process === 'undefined') {
+  // a poor node.js process shim - totally not great
+  var sys = require('system')
+  this.process = {
+    pid: sys.pid,
+    env: sys.env,
+    argv: sys.args.splice(),
+    stdin: sys.stdin,
+    stdout: sys.stdout,
+    stderr: sys.stderr
```

Comment:
This process shim needs to be created before trying to require a 3rd party reporter. Hmm a test should have failed for you here.... Coverage is missing.